sphinx/deprecation.py

```diff
@@ -56,10 +56,12 @@ def __getattr__(name: str) -> Any:
 
     qualified_name = f'{module}.{attribute}'
     if canonical_name:
-        message = (f'The alias {qualified_name!r} is deprecated, '
-                   f'use {canonical_name!r} instead.')
+        message = (
+            f'The alias {qualified_name!r} is deprecated, ' f'use {canonical_name!r} instead.'
```

Comment:
This one is confusing in the sense that two f-strings are concatenated on the same line.

Is there a way for ruff to merge the f-strings when possible?

Reply:
fixed manually

> Is there a way for ruff to merge the f-strings when possible?

It does not appear so, maybe a good feature request to open on ruff 😄